src/services/search.js

```diff
@@ -6,55 +6,61 @@ const { endpoints } = window;
 export async function queryIndexMapping(params) {
   const { indices } = params;
 
-  const endpoint = `${endpoints.elasticsearch}/${endpoints.prefix}${endpoints.run_index}${
-    indices[0]
-  }/_mappings`;
-  return request.get(endpoint);
+  const mappings = `${endpoints.prefix}${endpoints.run_index}${indices[0]}/_mappings`;
+
+  const endpoint = `${endpoints.pbench_server}/elasticsearch`;
+
+  return request.post(endpoint, { data: { indices: mappings } });
```

Comment:
Why not just call it `indices` instead of `mappings`? (I don't think the key "indices" is really any more accurate anywhere else than here, and it's just a name.)

But, no big deal, and no need to change it.